Application/ControlMsrE2/ControlMsrE2.c

```diff
@@ -0,0 +1,242 @@
+/** @file
+ Control CFGLock BIOS Option
+ 
+ Copyright (c) 2020, Brumbaer. All rights reserved.<BR>
+ This program and the accompanying materials
+ are licensed and made available under the terms and conditions of the BSD License
+ which accompanies this distribution.  The full text of the license may be found at
+ http://opensource.org/licenses/bsd-license.php
+ 
+ THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+ 
+ **/
+#include "ControlMsrE2.h"
+
+#define CONTEXTS_MAX 8
+
+EFI_STATUS WalkListHeaders (
+                            IN EFI_HII_HANDLE * handles,
+                            IN EFI_HII_PACKAGE_LIST_HEADER **ListHeaders,
+                            IN UINT32 ListHeaderCount,
+                            IN EFI_STRING SearchString
+                            ) {
+    UINT16           OptionsCount;
+    UINT16           ContextsCount;
+    OneOfContext     Contexts[CONTEXTS_MAX];
+    UINT32           ListHeaderIndex;
+    
+    EFI_STATUS Status = EFI_SUCCESS;
+    
+    OptionsCount = 0;
+    ContextsCount = 0;
+    ListHeaderIndex = 0;
+    
+    // For Each Handle
+
+    for (EFI_HII_HANDLE* h = (EFI_HII_HANDLE*) handles; (*h != NULL) && (ContextsCount < CONTEXTS_MAX); h++, ListHeaderIndex++) {
+
+        ListHeaders[ListHeaderIndex] = HiiExportPackageLists(*h);
+        
+        if (ListHeaders[ListHeaderIndex] != NULL) {
+            
+            if (IS_VERBOSE()) {
```

Comment:
For both functions and macros the argument braces should be padded with spaces, like `IS_VERBOSE ()` or `HiiExportPackageLists (*Handle)`.